### PR TITLE
✨ :sparkles: Add resizing

### PR DESCRIPTION
**Pull request by alexanderkasten**
## What did you change?

This PR adds a resize function for the propertypanel and has also a small style fix for form-controls.

## How can others test the changes?

- checkout the branch
- `au run`
- click the `resize` button and move your mouse for resizing the propertypanel
- click again to set the size

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: src/contracts/bpmnmodeler/constants.ts [hunk 1]
```diff
@@ -1,5 +1,9 @@
-
 export enum ElementDistributeOptions {
   HORIZONTAL = 'horizontal',
   VERTICAL = 'vertical',
 }
+
+export enum resizeOptions {
+  toggleBtn = 13,
```
Comment (5ebastianMeier) [inline src/contracts/bpmnmodeler/constants.ts]: irreführende Benennung - das scheinen mir Breiten zu sein - bitte versteck das nicht in einer Enum, sondern lagere das in die Konfiguration aus - falls es nicht konfigurierbar sein soll dann bitte in eine Konstante im ViewModel, in dem das genutzt wird

File: src/modules/bpmn-io/bpmn-io.scss [hunk 2]
```diff
@@ -44,8 +44,27 @@
 }
 
 .toggle_panel {
+  position: relative;
+  transform: rotate(-90deg);
+  bottom: 0;
+}
+
+.resize_btn {
```
Comment (heikomat) [inline src/modules/bpmn-io/bpmn-io.scss]: there is no real need to shorten `button` to `btn`

File: src/modules/bpmn-io/bpmn-io.ts [hunk 3]
```diff
@@ -11,16 +11,24 @@ import {ElementDistributeOptions,
         IShape} from '../../contracts/index';
 import environment from '../../environment';
 
+const toggleBtnWidth: number = 13;
```
Comment (heikomat) [inline src/modules/bpmn-io/bpmn-io.ts]: there is no real need to shorten `button` to `btn`

File: src/modules/bpmn-io/bpmn-io.ts [hunk 4]
```diff
@@ -11,16 +11,24 @@ import {ElementDistributeOptions,
...
+const resizeBtnWidth: number = 19;
```
Comment (heikomat) [inline src/modules/bpmn-io/bpmn-io.ts]: there is no real need to shorten `button` to `btn`

File: src/modules/bpmn-io/bpmn-io.ts [hunk 5]
```diff
@@ -11,16 +11,24 @@ import {ElementDistributeOptions,
...
+
 export class BpmnIo {
 
   private toggled: boolean = false;
   private toggleBtn: HTMLButtonElement;
+  private resizeBtn: HTMLButtonElement;
```
Comment (heikomat) [inline src/modules/bpmn-io/bpmn-io.ts]: there is no real need to shorten `button` to `btn`

File: src/modules/bpmn-io/bpmn-io.ts [hunk 6]
```diff
@@ -11,16 +11,24 @@ import {ElementDistributeOptions,
         IShape} from '../../contracts/index';
 import environment from '../../environment';
 
+const toggleBtnWidth: number = 13;
+const resizeBtnWidth: number = 19;
+
 export class BpmnIo {
 
   private toggled: boolean = false;
   private toggleBtn: HTMLButtonElement;
```
Comment (heikomat) [inline src/modules/bpmn-io/bpmn-io.ts]: there is no real need to shorten `button` to `btn`

File: src/modules/bpmn-io/bpmn-io.ts [hunk 7]
```diff
@@ -11,16 +11,24 @@ import {ElementDistributeOptions,
...
+  private resizeBtn: HTMLButtonElement;
   private panel: HTMLElement;
   private canvasModel: HTMLDivElement;
   private refresh: boolean = true;
+  private isResizeClicked: boolean = false;
 
-  private toggleBtnRight: string = '337px';
-  private canvasRight: string = '350px';
+  private toggleBtnRight: number = 337;
```
Comment (heikomat) [inline src/modules/bpmn-io/bpmn-io.ts]: there is no real need to shorten `button` to `btn`

File: src/modules/bpmn-io/bpmn-io.ts [hunk 8]
```diff
@@ -11,16 +11,24 @@ import {ElementDistributeOptions,
...
+  private resizeBtnRight: number = 331;
```
Comment (heikomat) [inline src/modules/bpmn-io/bpmn-io.ts]: there is no real need to shorten `button` to `btn`

File: src/modules/bpmn-io/bpmn-io.ts [hunk 9]
```diff
@@ -120,19 +128,48 @@ export class BpmnIo {
...
+      this.resizeBtn.style.right = '-18px';
       this.canvasModel.style.right = '1px';
       this.toggled = true;
     }
   }
+
+  public resize(): void {
+    this.isResizeClicked = true;
+    document.addEventListener('mousemove', (event: any) => {
+      if (this.isResizeClicked === true) {
```
Comment (heikomat) [inline src/modules/bpmn-io/bpmn-io.ts]: you can early-exit here:

```TypeScript
if (this.isResizeClicked === false) {
  return;
}
```

**Pull request merged**